test/widgets/content_test.dart

```diff
@@ -161,4 +161,33 @@ void main() {
       check(tester.takeException()).isA<AssertionError>();
     });
   });
+
+  group("CodeBlock", () {
+    Future<void> prepareContent(WidgetTester tester, String html) async {
+      await tester.pumpWidget(MaterialApp(home: BlockContentList(nodes: parseContent(html).nodes)));
+      await tester.pump();
+      await tester.pump();
```

Comment:
These latter pumps aren't needed:
```suggestion
      await tester.pumpWidget(MaterialApp(home: BlockContentList(nodes: parseContent(html).nodes)));
```

In the other `prepareContent`, they're needed because `GlobalStoreWidget` and `PerAccountStoreWidget` each involve an async step. (To read from the database and fetch from the server, respectively. Actually we go to some trouble to make `PerAccountStoreWidget` after the data for a given account is first loaded — see `_PerAccountStoreWidgetState.didChangeDependencies` and the test "PerAccountStoreWidget immediate data after first loaded" in test/widgets/store_test.dart — but on first load, both those async steps are pretty fundamentally needed.)

But here, there's no async load involved, so it all builds on the first frame. You can verify that by just deleting these and seeing that the tests still pass, thanks to those `find.text` expectations which ensure the tests wouldn't spuriously pass if the widgets we want to test weren't yet built.

Reply:
ack